starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaDrmBridge.java

```diff
@@ -464,7 +464,7 @@ public void onKeyStatusChange(
     mMediaDrm.setPropertyString("privacyMode", "disable");
     mMediaDrm.setPropertyString("sessionSharing", "enable");
     if (keySystem.equals("com.youtube.widevine.l3")
-        && mMediaDrm.getPropertyString("securityLevel") != "L3") {
+        && mMediaDrm.getPropertyString("securityLevel").equals("L3")) {
```

Comment:
Should it be `!mMediaDrm.getPropertyString("securityLevel").equals("L3")`?

Reply:
Done.  Thanks!